packages/dev/docs/pages/Support.mdx

```diff
@@ -12,10 +12,56 @@ export default Layout;
 
 ---
 category: Foundation
-description: How to contribute features, report bugs, or make suggestions to improve React Spectrum.
+description: How to get help with React Spectrum, how to report issues, and our development roadmap.
 keywords: [react spectrum support, react spectrum help]
 ---
 
 # Support
 
-For feedback, bug reports, general questions about React Spectrum, please visit https://github.com/adobe/react-spectrum/issues/.
+This page includes information about how to get help with React Spectrum, how to report issues, and our development roadmap.
+
+## Getting help
+
+The best place to ask questions about React Spectrum and get help is on Github. Please [file an issue](https://github.com/adobe/react-spectrum/issues)
```

Comment:
The best place to ask questions and get help about React Spectrum is on Github. Please [file an issue](https://github.com/adobe/react-spectrum/issues)